### DISCUSSION TITLE
Costs estimation for running Ant Media Server

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5289] [FAQs]
I'm looking to get a rough estimation with regards to running Ant Media Server in cluster mode.
Are there any tools or approaches available for conducting a rough cost analysis, taking into account variables like the number of broadcasts and viewers,  bitrate, deployment options (cloud or on-premise), server uptime, optimizing expenses, etc?

Having a rough estimation will help in making informed decisions and evaluating and optimizing costs for running Ant Media Server.

**Comment by akiziltoprak** [comment]
Thanks for the question.

We do have a cost calculator [here ](https://antmedia.io/cost-calculator/) that can help you to calculate your overall cost of using Ant Media Server in cluster mode. You can enter the number of concurrent broadcasts and viewers, Video Resolution that you need, hourly or monthly use of licenses preferred, or which cloud network you will use too or only on-premise use required. If you want to calculate the data consumption and cost on cloud networks, you can use any of those given including AWS, Azure, GCP, Linode, Digital Ocean and Alibaba. 
Hope it answers your question
